app/parse.py

```diff
@@ -8,8 +11,53 @@ class Quote:
     tags: list[str]
 
 
+BASE_URL = "https://quotes.toscrape.com/"
+QUOTES_FIELDS = [field.name for field in fields(Quote)]
+
+
+def parse_quote(quote: Tag) -> Quote:
+    return Quote(
+        text=quote.select_one("span.text").text,
+        author=quote.select_one("small.author").text,
+        tags=[tag.text for tag in quote.select_one(".tags").findAll(
+            "a", class_="tag"
+        )],
+    )
+
+
+def get_page_quotes(page_soup: Tag) -> list[Quote]:
+    quotes = page_soup.select(".quote")
+    return [parse_quote(quote) for quote in quotes]
+
+
+def get_current_page(url: str) -> BeautifulSoup:
+    response = requests.get(url).content
+    return BeautifulSoup(response, "html.parser")
+
+
+def get_quotes() -> [Quote]:
```

Comment:
The return type of the `get_quotes` function should be `list[Quote]` instead of `[Quote]`. This ensures consistency with type hinting and clarifies that the function returns a list of `Quote` objects.